### ISSUE TITLE
Inconsistent 'Settings' icon in the Post / Site editor and Widgets page

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#49989]
### Description

In the Post and Site editor, the 'Settings' panel (sidebar) icon has been changed from a cog icon to a drawer icon. See conversation here: https://github.com/WordPress/gutenberg/issues/40204#issuecomment-1276264434

<img width="408" alt="Screenshot 2023-04-21 at 16 03 19" src="https://user-images.githubusercontent.com/1682452/233657572-2328217f-f370-4cde-aa09-ca42284741da.png">

I'd have a few arguments against this change, as the new drawer icon only represents the visual aspect and placement of the settings and doesn't tell anything about what it is about. Also, the labelling of the button and of the panel is 'Settings' and I can't see how a drawer can communicate that concept. Visually, to me, the drawer icon is just two rectangles that don't tell me anything useful.

Apparently, the only reason for the change to the drawer icon is that the sidebar can contain other settings with a cog icon:

<img width="346" alt="Screenshot 2023-04-21 at 16 14 00" src="https://user-images.githubusercontent.com/1682452/233658815-c063278b-3ed1-4009-b14a-f7152c00ca40.png">

I'd argue those tabs should use visible text instead of icons, which would solve the top bar settings icon issue.

Regardless, I'd like to point out that in the Widgets page the Settings icon is still a cog:

<img width="1633" alt="Screenshot 2023-04-21 at 16 03 42" src="https://user-images.githubusercontent.com/1682452/233657945-1856fba0-1b4b-4678-9e58-29408ed87d0e.png">

Whatever the decision is, the top bar Settings icon should be consistent throughout the entire WordPress UI.

### Step-by-step reproduction instructions

- Activate a non-block based theme e.g. Twenty Twenty-One.
- Go to Appearance > Widgets.
- Observe the Settings icon in the top bar is still the 'cog' icon.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by joedolson** [comment]
I agree with everything @afercia says here. Obviously the key element of this issue is that the icons are inconsistent, and that needs to be fixed.

But I also fully agree that the sidebar icon is confusing, and the change wouldn't have been necessary if the sidebar tabs just used text labels - which would also help users understand what those tabs are for.

**Comment by t-hamano** [comment]
I believe the icon inconsistency was resolved in #51029. How about closing this issue and discussing whether the icon should be used or not in a new issue?

**Comment by afercia** [comment]
Thanks @t-hamano I'm going to close this issue. I see there is already https://github.com/WordPress/gutenberg/issues/46851 to discuss reverting back the icon to the cog.
